util/src/main/java/com/google/cloud/hadoop/util/CredentialFactory.java

```diff
@@ -202,7 +219,7 @@ protected TokenResponse executeRefreshToken() throws IOException {
    * Returns shared staticHttpTransport instance; initializes staticHttpTransport if it hasn't
    * already been initialized.
    */
-  private static synchronized HttpTransport getStaticHttpTransport()
+  public static synchronized HttpTransport getStaticHttpTransport()
```

Comment:
This method was used only to make request to local GCE metadata service. For external HTTP requests we need to use transport configured for GCS connector.